README.md

```diff
@@ -144,6 +144,7 @@ Usage of ./powd:
       --autocreatemasteraddr      Automatically creates & funds a master address if none is provided
       --debug                     Enable debug log level in all loggers.
       --devnet                    Indicate that will be running on an ephemeral devnet. --repopath will be autocleaned on exit.
+      --ffsusemasteraddr          Use the master address as the initial address for all new FFS instances instead of creating a new unique addess for each new FFS instance.
```

Comment:
I'm not completely sure about the name of the flag, but I can't figure out a better one.
Coming back always to would be good to have a more hierarchical order of flags, and being _file_ compatible too.

Reply:
Agree here.